fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -948,7 +948,7 @@ protected DruidWebService buildDruidWebService(DruidServiceConfig druidServiceCo
     protected Supplier<Map<String, String>> buildDruidWebServiceHeaderSupplier() {
         Supplier<Map<String, String>> supplier = HashMap::new;
         String customSupplierClassString = SYSTEM_CONFIG.getStringProperty(DRUID_HEADER_SUPPLIER_CLASS, null);
-        if (customSupplierClassString != null && customSupplierClassString.equals("")) {
+        if (customSupplierClassString != null && !customSupplierClassString.equals("")) {
```

Comment:
Probably, yes. But it doesn't simplify the expression, since you'd still need the `null` check. All I was concerned with for this fix was correcting the logic, rather than making the check more sound (I think `isEmpty` looks for al whitespace, of any kind, to consider it as empty)